tests/table/test_eval.py

```diff
@@ -241,15 +241,23 @@ def test_table_sign_bad_input() -> None:
 
 
 @pytest.mark.parametrize(
-    "fun",
+    ("fun", "is_arc"),
     [
-        "sin",
-        "cos",
-        "tan",
+        ("sin", False),
+        ("cos", False),
+        ("tan", False),
+        ("arcsin", True),
+        ("arccos", True),
+        ("arctan", True),
+        ("radians", False),
+        ("degrees", False),
     ],
 )
-def test_table_numeric_trigonometry(fun: str) -> None:
-    table = MicroPartition.from_pydict({"a": [0.0, math.pi, math.pi / 2, math.nan]})
+def test_table_numeric_trigonometry(fun: str, is_arc: bool) -> None:
+    if not is_arc:
+        table = MicroPartition.from_pydict({"a": [0.0, math.pi, math.pi / 2, math.nan]})
+    else:
+        table = MicroPartition.from_pydict({"a": [0.0, 1, 0.5, math.nan]})
```

Comment:
we should also have tests to ensure behavior for non valid values. A panic is probably fine!
I ran a test case on duckdb for `asin` for pi.
<img width="598" alt="image" src="https://github.com/Eventual-Inc/Daft/assets/2550285/8e7c7f0e-1706-4a49-85f5-b3da655c0f20">

Reply:
Fair enough. However, I think `nan` is more appropriate, because there may be only a few invalid values in a series at any one time, but panic will stop the world. You can always use something like `dropNan` to filter out invalid values if needed.

Reply:
That seems reasonable